src/pages/accounting/PeerDetailDrawer.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { useState } from 'react'
+import { Paper, Container, Drawer, Button, Typography, CircularProgress, Grid } from '@material-ui/core';
+import ClipboardCopy from '../../components/ClipboardCopy';
+import { beeDebugApi } from '../../services/bee';
+import EthereumAddress from '../../components/EthereumAddress';
+import { ConvertBalanceToBZZ } from '../../utils/common';
+
+function truncStringPortion(str: string, firstCharCount=10, endCharCount=10) {
+	var convertedStr="";
+	convertedStr+=str.substring(0, firstCharCount);
+	convertedStr += ".".repeat(3);
```

Comment:
Haha this is a bit excessive :). Also it should probably be a template string rather than operation on string